bundles/org.openhab.binding.haywardomnilogic/src/main/resources/OH-INF/thing/chlorinator.xml

```diff
@@ -67,41 +63,33 @@
 	</channel-type>
 
 	<channel-type id="chlorError">
-		<item-type>Number</item-type>
+		<item-type>String</item-type>
 		<label>Chlorinator Error</label>
-		<state readOnly="true"/>
+		<state pattern="%s" readOnly="true"/>
 	</channel-type>
 
 	<channel-type id="chlorAlert">
 		<item-type>String</item-type>
 		<label>Chlorinator Alert</label>
-		<description>Chlorinator Alert</description>
-		<state readOnly="true">
-			<options>
-				<option value="0">None</option>
-				<option value="16">Low T-Cell Temperature</option>
-			</options>
+		<state pattern="%s" readOnly="true">
```

Comment:
I'm not familiair with the API, but it would be great if somehow this could still have a set of options. 
I guess the list of possible String values is known/documented. If so, it would be nice to se eback the options. This would enable both proper UI support and i18n possibilities.

Reply:
Take a look at the README.  The alert, status and error channels are bit arrays that can have mulitple states turned on at any given time.  These are mostly for diagnostics.  If someone really wants to use them, I figured they could disect it in a rule, rather than me make these into a bunch of boolean/switch channels that most people won't use.

Reply:
Thanks